core/src/main/java/org/bitcoinj/crypto/BIP38PrivateKey.java

```diff
@@ -117,13 +117,13 @@ private BIP38PrivateKey(NetworkParameters params, byte[] bytes, boolean ecMultip
      * @return textual form
      */
     public String toBase58() {
-        return Base58.encodeChecked(1, bytes);
+        return Base58.encodeChecked(version, bytes);
     }
 
     public ECKey decrypt(String passphrase) throws BadPassphraseException {
         String normalizedPassphrase = Normalizer.normalize(passphrase, Normalizer.Form.NFC);
         ECKey key = ecMultiply ? decryptEC(normalizedPassphrase) : decryptNoEC(normalizedPassphrase);
-        Sha256Hash hash = Sha256Hash.twiceOf(key.toAddress(ScriptType.P2PKH, params.network()).toString().getBytes(StandardCharsets.US_ASCII));
+        Sha256Hash hash = Sha256Hash.twiceOf(key.toAddress(ScriptType.P2PKH, network).toString().getBytes(StandardCharsets.US_ASCII));
```

Comment:
This dependency on `network` has to be removed somehow if we want to remove the `network` member altogether.